docs/services/app_api.md

```diff
@@ -0,0 +1,76 @@
+# AppAPI
+
+For [AppAPI](https://github.com/cloud-py-api/app_api) the [Docker Socket Proxy](https://github.com/cloud-py-api/docker-socket-proxy) (DSP) is required to work.
+
+## HTTP AppAPI DSP
+
+### 1. Start the HTTP DSP container
+
+```bash
+docker compose up -d appapi-dsp
+```
+
+### 2. Configure Deploy daemon
+
+After the DSP container is running, configure the Deploy daemon in AppAPI admin settings with the following parameters:
```

Comment:
Can we also document which repo to clone for the app api settings to be available?

Reply:
Sure, but AppAPI is likely to be moved to server's core soon, so there will be no need to clone and build js, and now it's possible to just install AppAPI from AppStore, if you don't need the latest version from sources.